lib/zoom.js

```diff
@@ -25,7 +25,7 @@ export default (container, dimensions, scales, configuration, data, callback) =>
                 const drops = container.selectAll('.drop-line')
                     .selectAll('.drop')
                     .attr('cx',(d,i) => {
-                        return scalingFunction(new Date(d.date))
+                        return scalingFunction(configuration.date(d))
```

Comment:
I've copied the actual call from here, where it doesn't seem to need `new Date`:
https://github.com/marmelab/EventDrops/blob/master/lib/drawer/drops.js#L22